README.md

```diff
@@ -66,67 +69,69 @@ all this out you can go ahead and read the rest of this text.
 Contact
 -------
 
-You can chat with us on IRC in the channel [#home-manager][] on [OFTC][].
-There is also a [Matrix room](https://matrix.to/#/#hm:rycee.net),
-which is bridged to the IRC channel.
+You can chat with us on IRC in the channel [#home-manager][] on
+[OFTC][]. There is also a [Matrix
+room](https://matrix.to/#/#hm:rycee.net), which is bridged to the IRC
+channel.
 
 Installation
 ------------
 
 Home Manager can be used in three primary ways:
 
 1. Using the standalone `home-manager` tool. For platforms other than
-   NixOS and Darwin, this is the only available choice. It is also
-   recommended for people on NixOS or Darwin that want to manage their
-   home directory independently of the system as a whole. See
-   [Standalone installation][manual standalone install] in the manual
-   for instructions on how to perform this installation.
+   [NixOS][] and Darwin, this is the only available choice. It is also
+   recommended for people on [NixOS][] or Darwin that want to manage
+   their home directory independently of the system as a whole. See
+   [Standalone installation][manual standalone install] in the
+   [manual][] for instructions on how to perform this installation.
```

Comment:
I suspect that these added links will make it unclear to the user which link they actually want to use. Basically, there is only one link in this section that is relevant, the one that goes to the installation instructions and it would probably be best if that was the only thing that draws the user's attention.

Reply:
Same goes for 2 and 3 below.

Reply:
> I suspect that these added links will make it unclear to the user which link they actually want to use.

Thanks for the feeback! 19c3761958ae134927fe2091a25038b506fae96b should fix this.

> Basically, there is only one link in this section that is relevant, the one that goes to the installation instructions and it would probably be best if that was the only thing that draws the user's attention.

cfcc07502c644f10eb5c23e8596ad867d5e1cb91 addresses this by following the [Home Manager Manual](https://nix-community.github.io/home-manager/index.html#ch-installation) phrasing.